src/lime/extension/Main.hx

```diff
@@ -179,7 +179,17 @@ class Main {
 		
 		//var task = new Task (definition, description, "Lime");
 		var args = getCommandArguments (command);
-		var task = new Task (definition, args.join (" "), "lime");
+		var name = args.join (" ");
+		
+		var displayPort = getVshaxe ().displayPort;
+		if (getVshaxe ().enableCompilationServer && displayPort != null) {
+
+			args.push('--haxeflag="--connect $displayPort"');
```

Comment:
Doing this outside of `getCommandArguments()` to avoid this being shown as part of the name in the UI, which looks a bit strange.